ossapi/models.py

```diff
@@ -845,7 +846,43 @@ def override_types(self):
         type_ = BeatmapsetEventType(self.type)
         return {"comment": mapping[type_]}
 
+@dataclass
+class ChatChannel(Model):
+    channel_id: int
+    description: Optional[str]
+    icon: str
+    moderated: bool
+    name: str
+    type: ChannelType
+
+    # optional fields
+    # ---------------
+    first_message_id: Optional[int]
+    last_message_id: Optional[int]
+    last_read_id: Optional[int]
+    recent_messages: Optional[List[ChatMessage]]
+    users: Optional[List[int]]
+
+@dataclass
+class ChatMessage(Model):
+    channel_id: int
+    content: str
+    is_action: bool
+    message_id: int
+    sender: UserCompact
+    sender_id: int
+    timestamp: str
+
+@dataclass
+class CreateNewPMResponse(Model):
```

Comment:
This definitely needs a better name, but what? `CreateNewPM`?

Reply:
I think we can pare it down slightly to `CreatePMResponse` but other than that I don't have any bright ideas. Not the worst name in any case